hack/test-update-storage-objects.sh

```diff
@@ -107,16 +107,17 @@ echo "${ETCD_VERSION}/${STORAGE_BACKEND_ETCD2}" > "${ETCD_DIR}/version.txt"
 
 ### BEGIN TEST DEFINITION CUSTOMIZATION ###
 
+# After removing HPA in v1beta1 there is basically no resource which has
+# two versions. In the future we may want to extend it.
```

Comment:
Job has v1 and v2alpha1, can we test that? I feel it's unfair to ask you to test Job, but otherwise we don't test any version upgrades.

@soltysh it looks like you are the last one that touched this file, do you remember if we should test v1 and v2alpha1 Job in this script?